test/e2e_node/e2e_service.go

```diff
@@ -208,6 +202,45 @@ func (es *e2eService) startServer(cmd *healthCheckCommand) error {
 	return fmt.Errorf("Timeout waiting for service %s", cmd)
 }
 
+func (es *e2eService) stopService(name string) error {
+	// Since node services are started with sudo, we must get the PID of the child process.
+	out, err := exec.Command("pidof", name).Output()
```

Comment:
Using `-` only works on process groups (pgid). I'm not sure, but I think that might be the same as the test framework?
```console
stclair@node-e2e:~$ ps -eo pid,pgid,argsess h for help or q to quit)
  PID  PGID COMMAND
...
 9766  9729 sudo /tmp/gcloud-e2e-1774959747/kube-apiserver --etcd-servers http://127.0.0.1:4001 --insecure-bind-address 0.0.0.0 --service-cluster-ip-range 10.0.0.1/24 --kubelet-port 10250 --allow-privileged true --v 8 --logtostderr
 9767  9729 /tmp/gcloud-e2e-1774959747/kube-apiserver --etcd-servers http://127.0.0.1:4001 --insecure-bind-address 0.0.0.0 --service-cluster-ip-range 10.0.0.1/24 --kubelet-port 10250 --allow-privileged true --v 8 --logtostderr
 9771  9729 sudo /tmp/gcloud-e2e-1774959747/kubelet --api-servers http://127.0.0.1:8080 --address 0.0.0.0 --port 10250 --hostname-override node-e2e.us-central1-a.stclair-k8s-ext --volume-stats-agg-period 10s --allow-privileged true --ser
 9774  9729 /tmp/gcloud-e2e-1774959747/kubelet --api-servers http://127.0.0.1:8080 --address 0.0.0.0 --port 10250 --hostname-override node-e2e.us-central1-a.stclair-k8s-ext --volume-stats-agg-period 10s --allow-privileged true --serializ
 9783  9729 journalctl -k -f
...
```